File: 268-missing-number/268-missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        
        nums.sort()
        size=len(nums)+1
        
        if(0 not in nums):
            return 0
        for i in range(0,size-2):
            
            if((nums[i+1]-i)>1):
                return (nums[i+1]-1)
            
        return nums[size-2]+1
            
        

File: 268-missing-number/268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        unsigned long long int siz=nums.size();
        unsigned long long int ap=siz*(siz+1)/2;
        unsigned long long int sum=0;
        for(int i=0;i<nums.size();i++)
        {
            sum=sum+nums[i];
            
        }
        return ap-sum;
        
        

        
    }
};